pifpaf/__main__.py

```diff
@@ -52,17 +54,42 @@ def get_parser(self, prog_name):
                                 help="command to run")
             return parser
 
+        def _format_multiple_exceptions(self, e):
```

Comment:
That seems to be static. We could also probably define it on a base class used by RunDaemon once and for all, just for the sake of not redefiniting the same static method over and over again and saving memory.